platform/cert/cert_manager.go

```diff
@@ -160,6 +173,13 @@ func (c *certManager) UpdateCertificates(certs string) error {
 		return bosherr.WrapError(err, "Running command to update certificates without retries")
 	}
 
+	if strings.Compare(c.logTag, "PhotonOSCertManager") == 0 {
```

Comment:
Can you please change this to `c.logTag == "PhotonOSCertManager"` since it is clearer and more performant; however, we're wondering if the `c.logTag` is the best way to detect we are on PhotonOS here -- can you please explain that decision?

Reply:
Inside the update certificate function call, I found this a rather simply way to distinctly identify the platform. Also PhotonOSCertManager logtag wont be used by other Distribution.
Please correct me if I am wrong.

Reply:
We'd like to propose an alternative approach:
1. Write a script that calls `/usr/bin/c_rehash` and cats all the ca certificates into the `/etc/pki/tls/certs/ca-bundle.crt` file
2. Use that script as the `updateCmdPath` argument.
3. Remove the now-unnecessary command and logTag-testing conditional since the script has taken care of everything necessary for setting up PhotonOS.

Reply:
How would that script be distributed? The `updateCmdPath` seems to be primarily used for binaries that are built into the OS to perform CA certificate updates.

Reply:
Preferably we would like to have a PhotonOS stemcell native script that includes the ca certificate `cat` command, e.g. /usr/bin/photonos_cert_updater. This way we can maintain the old certmanager update certificate logic and simply call a PhotonOs-specific updateCmdPath. 

This script should be baked in the Photon OS stemcell.